opacus/accountants/rdp.py

```diff
@@ -30,6 +30,9 @@ def step(self, noise_multiplier: float, sample_rate: float):
     def get_privacy_spent(
         self, delta: float, alphas: Optional[List[Union[float, int]]] = None
     ) -> Tuple[float, float]:
+        if not self.steps:
+            return 0, 0
+
```

Comment:
If you call `get_privacy_spent` on an empty accountant, it fails with a cryptic exception. 
better to return 0